source/jormungandr/jormungandr/scenarios/experimental.py

```diff
@@ -575,192 +42,124 @@ def __init__(self):
+                                                             request=request)
+
+        pt_journey_pool = PtJourneyPool(instance=instance,
+                                        requested_orig_obj=requested_orig_obj,
+                                        requested_dest_obj=requested_dest_obj,
+                                        streetnetwork_path_pool=streetnetwork_path_pool,
+                                        krakens_call=krakens_call,
+                                        orig_fallback_durations_pool=orig_fallback_durations_pool,
+                                        dest_fallback_durations_pool=dest_fallback_durations_pool,
+                                        request=request)
+
+        completed_pt_journeys = wait_and_complete_pt_journey(requested_orig_obj=requested_orig_obj,
+                                                             requested_dest_obj=requested_dest_obj,
+                                                             pt_journey_pool=pt_journey_pool,
+                                                             streetnetwork_path_pool=streetnetwork_path_pool,
+                                                             orig_places_free_access=orig_places_free_access,
+                                                             dest_places_free_access=dest_places_free_access,
+                                                             orig_fallback_durations_pool=orig_fallback_durations_pool,
+                                                             dest_fallback_durations_pool=dest_fallback_durations_pool,
+                                                             request=request)
+
+        for mode in requested_dep_modes:
+            dp = streetnetwork_path_pool.wait_and_get(requested_orig_obj=requested_orig_obj,
+                                                      requested_dest_obj=requested_dest_obj,
+                                                      mode=mode,
+                                                      period_extremity=period_extremity,
+                                                      streetnetwork_path_type=StreetNetworkPathType.DIRECT)
+            if getattr(dp, "journeys", None):
+                res.append(dp)
+
+        res.extend([j for j in completed_pt_journeys if j])
+
+        check_final_results_or_raise(res, orig_fallback_durations_pool, dest_fallback_durations_pool)
+
+        for r in res:
             fill_uris(r)
-        return resp
-
-    def isochrone(self, request, instance):
-        return self.__on_journeys(type_pb2.ISOCHRONE, request, instance)
+        return res
 
-    def journeys(self, request, instance):
+    def call_kraken(self, request_type, request, instance, krakens_call):
         logger = logging.getLogger(__name__)
-        logger.warn('using experimental scenario!!!')
-        _init_g()
-        return self.__on_journeys(type_pb2.PLANNER, request, instance)
+        logger.warning("using experimental scenario!!")
```

Comment:
Ok, i'll create another PR to rename the scenario so that we have a milestone PR to mark the historical moment :)